app/models/shelter.rb

```diff
@@ -31,4 +32,24 @@ def alphabetical_pets
   def shelter_pets_filtered_by_age(age_filter)
     adoptable_pets.where("age >= ?", age_filter)
   end
+
+  def self.pending_applications
```

Comment:
Interesting method here! I'd be curious if we can do this using joins instead of enumerables.  Might be something we can get feedback on tomorrow from the mentor.